mrp_unbuild_cust_alu_analytics/readme/CONTRIBUTORS.rst

```diff
@@ -1,3 +1,4 @@
 * Adrian Resua <adrian.resua@solvos.es>
 * David Alonso <david.alonso@solvos.es>
 * Christian Santamaria <christian.santamaria@solvos.es>
+* Ángel Rivas <angel.rivas@solvos.es>
```

Comment:
Once you've added yourself to contributors, re-creating `README.rst` is required.

Reply:
@dalonsod Done!